cypress/e2e/API Tests/getCustomers.spec.js

```diff
@@ -0,0 +1,14 @@
+const API_URL = Cypress.env('API_URL')
+const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`
+
+describe('API Test - Return customers', () => {
```

Comment:
Será que a descrição da suíte precisa mesmo desta parte visto que o teste já está na pasta `API Tests/`?

A propósito, quando criar pastas, recomendo **não** deixar espaços em branco, eu iria só de `API/` e `GUI/`.

```suggestion
describe('Return customers', () => {
```

A propósito, `Return customers` encaixa mais para o nome do caso de teste, não para o nome da suíte.

Pra mim, a suíte deveria ser chamada `EngageSphere`, visto que são os testes de API para tal aplicação.